auto_py_to_exe/web/css/general.css

```diff
@@ -32,6 +29,19 @@
     --border-radius: 4px;
 }
 
+.dark-theme {
+    --background: #15131E;
+    --unselected: #5f5f5f;
+    --text: #ffffff;
+
+    --title: #e2e2e2;
+}
+
+* {
+    box-sizing: border-box;
+    color: var(--text);
```

Comment:
This is now setting the "Help Post" link an incorrect colour

Reply:
I think this happens because main.css provides styles directly on the 'a' tag, so I wrapped the "Github" text in a span for uniformity.